grommet/grommet
# Add defaultProps import

Signed-off-by: Martin Mecir mecir.martin@gmail.com

Related to #5229

What does this PR do?

Like other Grommet components, DataTable will default to defaultProps.theme if there's no Theme context available. However defaultProps isn't imported in the Body component inside DataTable component. This PR adds that missing import.

Where should the reviewer start?

Line 14 is the change; line 158 is the reference to defaultProps

How should this be manually tested?

You could use a DataTable outside of a ThemeContext -- it will fail without this change.

Any background context you want to provide?

I hit this bug while trying to reproduce the Datatable example from `grommet-sandbox` in separate project. I used it as standalone component and that probably broke ThemeContext enough that my DataTable tried to fall back to (the non-existent) defaultProps.

Do the grommet docs need to be updated?

Nope

Should this PR be mentioned in the release notes?

Not unless you list every bugfix.

Is this change backwards compatible or is it a breaking change?

Compatible.
